### PR TITLE
BGDIINF_SB-2530: add HTTP readiness probe -  -  -  -  - #minor

### PR DESCRIPTION
**Pull request by ltclm**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -16,6 +18,26 @@ def checker():
...
 
 
+@app.route('/checker/ready', methods=['GET'])
+def readiness():
+    sphinx_ok_string = 'READY\n'
+    try:
+        with open(SPHINX_BACKEND_READY, 'r', encoding='utf-8') as fd:
+            content = fd.read()
+    except IOError:
+        content = ""
```
Comment (ltshb) [inline app/routes.py:28]: I would log the error here for debugging and might be a good idea to use a critical level.

File: app/routes.py [hunk 2]
```diff
@@ -16,6 +18,26 @@ def checker():
...
+
+    if content != sphinx_ok_string:
+        logger.error(
+            'Incomprehensible sphinx backend %s answer: %s. '
+            'sphinx search service is probably not ready yet.',
+            SPHINX_BACKEND_READY,
+            content,
+        )
```
Comment (ltshb) [inline app/routes.py:36]: The log here is not necessary as the `abort()` will be already logged here https://github.com/geoadmin/service-search-wsgi/blob/a8055ecb2f59d0ec7af8d4fa7e62d332a7eedfcc/app/__init__.py#L93

File: app/settings.py [hunk 3]
```diff
@@ -100,3 +100,7 @@
 GZIP_COMPRESSION_LEVEL = int(os.getenv('GZIP_COMPRESSION_LEVEL', '9'))
 
 WSGI_TIMEOUT = int(os.getenv('WSGI_TIMEOUT', '1'))
+
+# readiness probe
+PROBE_MOUNTPOINT = os.getenv('PROBE_MOUNTPOINT', '/tmp')
```
Comment (ltshb) [inline app/settings.py:105]: Add the `PROBE_MOUNTPOINT` to the README.md

**Comment by ltclm** [comment]
@ltshb thank you for the feedback, i have addressed your points.

**Review by ltshb — APPROVED** [review]
👍🏼

**Comment by ltshb** [comment]
Oops unittest not working

File: app/routes.py [hunk 4]
```diff
@@ -16,6 +18,21 @@ def checker():
     return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
 
 
+@app.route('/checker/ready', methods=['GET'])
+def readiness():
+    sphinx_ok_string = 'READY\n'
+    try:
+        with open(SPHINX_BACKEND_READY, 'r', encoding='utf-8') as fd:
+            content = fd.read()
+    except IOError as e:
```
Comment (ltshb) [inline app/routes.py:27]: lint doesn't like `e` use `err` or `error` instead.

**Review by ltshb — APPROVED** [review]

**Pull request merged**